src/inmanta/data/dataview.py

```diff
@@ -1211,16 +1213,19 @@ def get_base_url(self) -> str:
 
     def get_base_query(self) -> SimpleQueryBuilder:
         base = SimpleQueryBuilder(
-            select_clause="""SELECT a.name, a.environment, last_failover, paused, unpause_on_resume,
-                                            ap.hostname as process_name, ai.process as process_id,
-                                            (CASE WHEN paused THEN 'paused'
-                                                WHEN id_primary IS NOT NULL THEN 'up'
-                                                ELSE 'down'
+            select_clause="""SELECT a.name, a.environment, a.last_failover, a.paused, a.unpause_on_resume,
```

Comment:
Could you clean up the formatting a bit while you're at it, to improve readability ?